PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+PULL REQUEST TEMPLATE
+======================
```

Comment:
just to check but these lines are dropped when the template is used right?

Reply:
removed